crates/bevy_math/src/ops.rs

```diff
@@ -21,121 +21,208 @@
 
 #[cfg(not(feature = "libm"))]
 mod std_ops {
+
+    /// Raises a number to a floating point power.
+    ///
+    /// Specified or Unspecified Precision is determined by whether the `libm` feature is enabled.
```

Comment:
what about just "Precision is specified when the `libm` feature is enabled."?
i think that's clearer to me.

Reply:
done, much nicer phrasing.